react-native/react/common-adapters/checkbox.js.flow

```diff
@@ -5,8 +5,8 @@ import React, {Component} from '../base-react'
 export type Props = {
   label: string,
   onCheck: Function,
-  checked: bool,
-  style: object
+  checked: boolean,
```

Comment:
it should be boolean according to the docs (http://flowtype.org/docs/quick-reference.html#built-in-primitive-types) but they use both in the flow code.